ui/src/main/webapp/src/app/components/project-list/no-projects-welcome.component.html

```diff
@@ -1,13 +1,13 @@
 <div class="blank-slate-pf">
-    <h1>
-        Welcome to the Red Hat Application Migration Toolkit Web Console
+    <h1 i18n>
+        Welcome to the Red Hat Application Migration Toolkit Web Console.
     </h1>
     <div class="blank-slate-pf-main-action">
-        <a (click)="newProjectClicked()" class="clickable btn btn-primary btn-lg">New Project</a>
+        <a (click)="newProjectClicked()" class="clickable btn btn-primary btn-lg" i18n="Button">New Project</a>
     </div>
 
-    <div class="welcome-help-text">
-        To learn more, visit the <a href="https://github.com/windup/windup-web/blob/master/README.md">
-            Red Hat Application Migration Toolkit Web Console Documentation</a>.
+    <div class="welcome-help-text"><template i18n="1st part">
```

Comment:
I think this one deserves some more descriptive text - it is not very clear what 1st part means. 

Translator will see text in format similar to this one:
```xml
      <trans-unit id="d2a7deb7096cada2c3177157652680fe7e6de0ed" datatype="html">
        <source>To learn more, visit the</source>
        <target/>
        <note priority="1" from="description">1st part</note>
      </trans-unit>
``` 
it doesn't say much about the context. The same for 2nd part.